docs/metasploit-framework.wiki/Attacking-ADCS-ESC-Vulnerabilities.md

```diff
@@ -0,0 +1,931 @@
+# Setting Up An AD CS Target
```

Comment:
This file should be named `Attacking-AD-CS-ESC-Vulnerabilities.md` right with the `-` in `AD-CS`?

Reply:
This file was created before we did the renaming, I can go ahead and fix this now though but a note per speaking with Alan that we will need to revisit this point and fix up other instances still once the Kerberos branch lands.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/17399/commits/239bc42f4c4228a21dd726e08fa3273545112f7f